Pa3/Durchführung/Project/app/EventEnum.h

```diff
@@ -21,20 +21,23 @@
 */
 typedef enum
 {
-  INIZALIZATION_DONE
-  CALIBRATION_DONE
-  COUNTDOWN_IS_FINISHED
-  POWER_TO_THE_MOTORS_HAS_BEEN_STOPPED
-  START_BUTTON_HAS_BEEN_RELEASED
-  CALIBRATION_BUTTON_HAS_BEEN_RELEASED
-  PARAM_BUTTON_HAS_BEEN_RELEASED
-  START_FINISH_LINE_WAS_RECOGINZED
-  THE_GUIDELINE_WAS_RECOGNIZED
-  GUIDELINE_WAS_LOST
-  LAPTIME_IS_TOO_LONG
-  DRIVE_OVER_GAP_IS_ACTIVE_FOR_TOO_LONG
+  NO_EVENT_HAS_HAPPEND = 0,
+  //Normal events
```

Comment:
See coding style guide for non-doxygen comment style
```suggestion
  /*Normal events*/
```